propnet/core/tests/test_mp_interface.py

```diff
@@ -0,0 +1,13 @@
+import unittest
+from propnet.core.symbols import *
+from propnet.symbols import *
+from propnet.ext.matproj import *
+
+class MPInterfaceTest(unittest.TestCase):
+    """ """
+
+    def testLoadProperties(self):
+        mpID = 'mp-1153'
```

Comment:
no capital letters in variable names please https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

Reply:
got you -- I wasn't sure what to do since the methods in test suites seem to be following another capitalization scheme? I'll just go through and change all camel case to underscores.

Reply:
ie. testAllProperties method, assertEqual method, etc.

Reply:
yes, don't ask me why that is .. PEP-8 is the official rulebook, the unittest methods are a bit of a historical anomaly .. I think (stackoverflow informs me) that it's because it was originally a direct port of some Java code, but who knows